includes/pages/admin_shifts.php

```diff
@@ -339,9 +339,9 @@ function admin_shifts()
                 $shifts_table_entry = [
                     'timeslot'      =>
                         icon('clock-history') . ' '
-                        . $start->format(__('general.datetime'))
+                        . dateWithEventDay($start->format('Y-m-d')) . ' ' . $start->format(__('H:i'))
                         . ' - '
-                        . '<span title="' . $end->format(__('general.date')) . '">'
+                        . '<span title="' . dateWithEventDay($end->format('general.date')) . '">'
```

Comment:
Please test your changes (also in both languages) ;)

`general.date` is a translation id and not a valid date format :P